.gitlab/lassen-jobs.yml

```diff
@@ -80,13 +80,13 @@ pgi_19_10_cuda:
 
 pgi_19_10_gcc_8_3_1_cuda:
   variables:
-    SPEC: "+fortran+cuda %pgi@19.10 cxxflags=--gcc-toolchain=/usr/tce/packages/gcc/gcc-8.3.1 cflags=--gcc-toolchain=/usr/tce/packages/gcc/gcc-8.3.1 ^cuda@10.1.168"
+    SPEC: "+fortran+cuda %pgi@19.10 cxxflags=--gcc-toolchain=/usr/tce/packages/gcc/gcc-8.3.1 cflags=--gcc-toolchain=/usr/tce/packages/gcc/gcc-8.3.1 cuda_arch=70 ^cuda@11.0.2"
```

Comment:
Can this be CUDA 10 again?

Reply:
Is this a request to change it back to CUDA 10 or just whether it can be?

Reply:
A request lol at one point in time, I thought I needed to change these to cuda 11 so I could use the "--forward-unknown-to-host-compiler" flag so that nvcc would stop complaining about not knowing what PGI's "-Minform=severe" flag is. However, later on we went back and changed the way that I pass the PGI flags in and it become unnecessary to keep around.

Reply:
and I'll probably also undo the cuda_arch change too. That should be part of a separate PR that has the device allocator improvements.